samples/react/counter/app.fsproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>netstandard1.6</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="../node_modules/fable-elmish-react/Fable.Elmish.React.fsproj" />
+    <ProjectReference Include="../../../src/elmish-react/Fable.Elmish.React.fsproj" />
```

Comment:
I like samples using npm dependencies - tells me that the stuff I published works.